docs/markdown/authoritative/settings.md

```diff
@@ -635,6 +635,15 @@ method to distribute the zone data to the slaves), then set `only-notify` to an
 value and specify the notification targets explicitly using [`also-notify`](#also-notify)
 and/or `ALSO-NOTIFY` domain metadata to avoid this potential bottleneck.
 
+## `enable-notification-warning`
+* Boolean
+* Default: yes
+
+By default PowerDNS will log a warning if all NOTIFYs were suppressed due to
+`only-notify` and there were no explict ALSO-NOTIFYs. This warning can be suppressed by
+setting this option to `no`. Disabling the warning may be useful with `slave-renotify=yes`
+and empty `only-notify`, but only a few slave zones have ALSO-NOTIFYs configured.
```

Comment:
You are right, The last sentence is confusing for everybody except me. So i removed it.